### DISCUSSION TITLE
Waka readme not updating automatically.

### DISCUSSION THREAD

**Discussion by chitraarasu** [athul/waka-readme#87] [Q&A] [answered]
![Screenshot 2022-08-06 085614](https://user-images.githubusercontent.com/75994428/183231880-53996c39-c805-4b48-9e33-ba880c726d87.png)

## So far it has worked perfectly. Since last week it is not run automatically. it is failed. If I run it manually it works perfectly. Is there any way to fix this?

![Screenshot 2022-08-06 090440](https://user-images.githubusercontent.com/75994428/183232021-631d4f34-eb8a-413c-8b2b-2e9a04e7aff1.png)

## It is my code.

![Screenshot 2022-08-06 090936](https://user-images.githubusercontent.com/75994428/183232182-9a331c6b-e82b-46c4-b7db-1d82f9a000be.png)

**Comment by nandgator** [comment, marked as answer]
Updated: https://github.com/athul/waka-readme/issues/85#issuecomment-1207163776
